dist/common/scripts/scylla_util.py

```diff
@@ -120,19 +120,19 @@ def __init__(self):
 
     @staticmethod
     def is_gce_instance():
-      """Check if it's GCE instance via DNS lookup to metadata server."""
-      import socket
-      try:
-        addrlist = socket.getaddrinfo('metadata.google.internal', 80)
-      except socket.gaierror:
+        """Check if it's GCE instance via DNS lookup to metadata server."""
+        import socket
+        try:
```

Comment:
This commit should be folded into the commit that introduced the problem. We try not to introduce problems and fix them in the same series.

Reply:
would it make then sense to squash everything into original 

commit b93284b00197526e894d9f83ce8ff9e3d05f8175
Author: Lubos Kosco <tarzanek@gmail.com>
Date:   Mon Jun 22 16:04:22 2020 +0200

    add out of box configs for GCP VMs with nvmes

    not recommended setups will still run iotune
    fixes #6631

?

because that commit basically says it all :-)
I am kinda confused now with want we want to achieve with the commits and their messages.

I understand we want:
1) have a clear message in the commit about what it does
2) be able to review fixes to original commit (do we care?)
3) have a sane commits for backports

hence the suggestion to fold everything into original commit (and have just the refactor iotune commit and adding the supported/recommended configs)

Reply:
of course the message would be:
```
scylla_io_setup: add out of box configs for GCP VMs with nvmes

not recommended setups will still run iotune
fixes #6631
```
with the subsystem prefix (if prefix isn't free form, some LOV should define it)

Reply:
I'll have a look at other messages used in repo

Reply:
> would it make then sense to squash everything into original
> 
> commit [b93284b](https://github.com/scylladb/scylla/commit/b93284b00197526e894d9f83ce8ff9e3d05f8175)
> Author: Lubos Kosco [tarzanek@gmail.com](mailto:tarzanek@gmail.com)
> Date: Mon Jun 22 16:04:22 2020 +0200
> 
> ```
> add out of box configs for GCP VMs with nvmes
> 
> not recommended setups will still run iotune
> fixes #6631
> ```
> 
> ?
> 
> because that commit basically says it all :-)
> I am kinda confused now with want we want to achieve with the commits and their messages.
> 
> I understand we want:
> 
> 1. have a clear message in the commit about what it does
> 2. be able to review fixes to original commit (do we care?)
> 3. have a sane commits for backports
> 
> hence the suggestion to fold everything into original commit (and have just the refactor iotune commit and adding the supported/recommended configs)

A huge patch, especially with a two-line description, is very hard to review.

If you move something out and then make a small change, it is much easier to review as two patches. For the first patch that moves things, I make a cursory check to ensure that's all (or, if I have prior good experience with the contributor, I just skip it). The changes in the second patch then stand out from the noise of the code movement and can be reviewed in isolation.

Reply:
I guess my conflict is mainly the continuity of review process
since you saw the original patch
and the new commits were mainly the resolved noise from first patch (which all made sense and I did fix them hopefully to satisfy all reqs)

but I guess I should more than the noise worry about first patch (and as you said fold fixes into first patch, which in this case it was all of them)